app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,33 @@
-<h2>Log in</h2>
+</html> 
```

Comment:
No need for an `html` tag, if you can recall, this directly gets inserted into the layout template (application.html.erb), so this would result in having 2 `html` tags